pkg/logs/wait_for_image.go

```diff
@@ -173,3 +178,16 @@ func filterErrors(condition watchTools.ConditionFunc) watchTools.ConditionFunc {
 		return condition(event)
 	}
 }
+
+func (w *imageWaiter) imageBuildStarted(ctx context.Context, namespace, buildName string) error {
+	build, err := w.KpackClient.KpackV1alpha1().Builds(namespace).Get(ctx, buildName, v1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	condition := build.Status.GetCondition(corev1alpha1.ConditionSucceeded)
+	if condition.IsFalse() {
+		return errors.New(condition.Message)
```

Comment:
```suggestion
		return buildFailure(condition.Message)
```

https://github.com/buildpacks-community/kpack/blob/ca60665cc2f228d6054dd00eacb721d69aa9bb54/pkg/logs/wait_for_image.go#L137-L144